Mollier/SAM.Core.Mollier.UI/Forms/MollierForm.cs

```diff
@@ -560,12 +575,16 @@ private void CheckBox_Zone_CheckedChanged(object sender, EventArgs e)
                 List<UIMollierZone> mollierZones = Query.MollierZones();
                 foreach (UIMollierZone zone in mollierZones)
```

Comment:
Could we just add the full list?

public List<IUIMollierObject> AddMollierObjects<T>(IEnumerable<T> mollierObjects, bool checkPressure = true)